src/main/java/com/codessquad/qna/domain/Answer.java

```diff
@@ -0,0 +1,47 @@
+package com.codessquad.qna.domain;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+@Entity
+public class Answer {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User writer;
+
+    @ManyToOne
+    @JoinColumn(name = "question_id")
+    private Question question;
+
+    private String contents;
+    private LocalDateTime createDate;
```

Comment:
```suggestion
    private LocalDateTime createDateTime;
```

DateTime 형이므로 이름에 그것이 드러나도록 해주는 것이 좋을 것 같습니다.

Reply:
수정했습니다!